UIforETW/UIforETWDlg.cpp

```diff
@@ -1131,7 +1940,7 @@ void CUIforETWDlg::OnSize(UINT nType, int /*cx*/, int /*cy*/)
 		int yDelta = windowRect.Height() - lastHeight_;
 		lastHeight_ += yDelta;
 
-		UINT flags = SWP_NOMOVE | SWP_NOOWNERZORDER | SWP_NOZORDER | SWP_NOACTIVATE;
+		const UINT flags = (SWP_NOMOVE | SWP_NOOWNERZORDER | SWP_NOZORDER | SWP_NOACTIVATE);

```

Comment:
- bitwise operators have strange precedence rules
- primary expressions are more obviously correct/easier for the reader to reason about
- parenthesis are cheap

...other than that, they're unnecessary.